python_tse/rdv_resumo.py

```diff
@@ -28,7 +28,8 @@ class Fase(asn1.Enumerated):
     _map = {
         1: "simulado",
         2: "oficial",
-        3: "treinamento",
+        3: "treinamento",if voto13t
+Than 22
```

Comment:
Alterar para 26. 22 não vai funcionar em 2026, vide https://docs.python.org/3/tutorial/controlflow.html

Reply:
Caso não haja partido, sugiro utilizar _

Reply:
Seguindo as dicas do @Doctor-God e as dicas do Clean Code do Bob Martin, seria interessante colocar o `13` em uma variavel e então multiplicá-la por 2 após o `Than`

Quando for 2039 é só multiplicar por 3 e assim por diante